WordPress/Classes/ViewRelated/Blog/SettingTableViewCell.h

```diff
@@ -0,0 +1,5 @@
+#import "WPTableViewCell.h"
+
+@interface SettingTableViewCell : WPTableViewCell
```

Comment:
Mixed feelings around here, since this represents a single setting, not a group of settings, and singular actually makes sense.